tests/integration/test_multiple_disks/test.py

```diff
@@ -632,12 +632,13 @@ def get_paths_for_partition_from_part_log(node, table, partition_id):
     return paths.strip().split('\n')
 
 
-@pytest.mark.parametrize("name,engine", [
-    pytest.param("altering_mt", "MergeTree()", id="mt"),
+@pytest.mark.parametrize("name,engine,use_metadata_cache", [
+    pytest.param("altering_mt", "MergeTree()", "false", id="mt"),
+    pytest.param("altering_mt", "MergeTree()", "true", id="mt_use_metadata_cache"),
```

Comment:
It will great to have a test with corrupted checksum, so that `CHECK TABLE` will show the problem, I guess removing the cache just before `CHECK TABLE` may be enough. What do you think?

Reply:
Good idea, I will try.

Reply:
I've tried few things but this does not work:
- rm some files in rocksdb folder (they are opened, does not work)
- truncate some files in rocksdb folder (they are in cache, does not work)
- but truncate will make db invalid, and so on restart open will fail

So I guess the only option is to replace the value, or ignore this for now.

Reply:
Do you think it is necessary to add a settings like 'write_corrupted_metadata_cache', when it is set true, insert bad value into metadata cache. But I think it is too tricky... 
Maybe ignore for now is a better choice.

Reply:
>Maybe ignore for now is a better choice.

Completely agree.